packages/client/components/AddTeamDialog.tsx

```diff
@@ -0,0 +1,201 @@
+      setTeamName(generateTeamName(newUsers))
+    }
+  }
+
+  const handleAddTeam = () => {
+    const newTeam = {
+      name: teamName.substring(0, MAX_TEAM_NAME_LENGTH),
+      orgId: selectedOrgId
+    }
+    submitMutation()
+
+    AddTeamMutation(
+      atmosphere,
+      {newTeam, invitees: selectedUsers.map((user) => user.email)},
+      {
+        onError,
+        onCompleted: (res, errors) => {
+          onCompleted(res, errors)
+          const error = getGraphQLError(res, errors)
+          if (!error) {
+            onAddTeam(res.addTeam.team.id)
+          }
+        },
+        history,
+        showTeamCreatedToast: false
+      }
+    )
+  }
+
+  const isValid = selectedUsers.length && teamName.trim()
+
+  return (
+    <Dialog isOpen={isOpen} onClose={onClose}>
+      <DialogContent className='z-10'>
+        <DialogTitle className='mb-4'>Add team</DialogTitle>
+
+        <fieldset className={fieldsetStyles}>
+          <label className={labelStyles}>Add teammates</label>
+
+          <AdhocTeamMultiSelect
+            viewerRef={viewer}
+            value={selectedUsers}
+            onChange={onSelectedUsersChange}
+          />
+
+          {selectedUsers.some((user: Option) => !user.id) && (
+            <div className='mt-3 text-xs font-semibold text-slate-700'>
+              Email invitations expire in 30 days.
+            </div>
+          )}
+        </fieldset>
+
+        {showOrgPicker && (
+          <fieldset className={fieldsetStyles}>
+            <label className={labelStyles}>Organization</label>
+            <Select onValueChange={(orgId) => setSelectedOrgId(orgId)} value={selectedOrgId}>
+              <SelectTrigger className='bg-white'>
+                <SelectValue />
+              </SelectTrigger>
+              <SelectContent>
+                <SelectGroup>
+                  {viewerOrganizations
+                    .filter((org) => (mutualOrgsIds.length ? mutualOrgsIds.includes(org.id) : true))
+                    .map((org) => (
+                      <SelectItem value={org.id} key={org.id}>
+                        {org.name}
+                      </SelectItem>
+                    ))}
+                </SelectGroup>
+              </SelectContent>
+            </Select>
+          </fieldset>
+        )}
+
+        <fieldset className={fieldsetStyles}>
+          <label className={labelStyles}>Team name</label>
+          <Input
+            onChange={(e) => {
+              !teamNameManuallyEdited && setTeamNameManuallyEdited(true)
+              setTeamName(e.target.value)
+            }}
+            value={teamName}
+          />
+        </fieldset>
+        {error && <div className='mt-2 text-sm font-semibold text-tomato-500'>{error.message}</div>}
+        <DialogActions>
+          <SecondaryButton onClick={onClose} size='small'>
+            Cancel
+          </SecondaryButton>
+          <PrimaryButton size='small' onClick={handleAddTeam} disabled={submitting || !isValid}>
```

Comment:
+1 I'd prefer the buttons to be a bit larger as they feel slightly small to me: 

![Screenshot 2023-09-28 at 17 01 24](https://github.com/ParabolInc/parabol/assets/39854876/8fc9b9d3-02c5-4b0e-9e13-535f15da75b1)

Reply:
made bigger

<img width="706" alt="image" src="https://github.com/ParabolInc/parabol/assets/466991/e20b8707-a3ec-4c13-86e0-54141021f457">